### ISSUE TITLE
SyntaxError on first run

### ISSUE THREAD

**Issue by queltosh** [xgi/castero#32]
After a brand new install via pip3 or source code i get this error:

```
root@26a84df338cf:~# pip3 install castero
Requirement already satisfied: castero in /usr/local/lib/python3.5/dist-packages/castero-0.5.3-py3.5.egg
Requirement already satisfied: beautifulsoup4 in /usr/local/lib/python3.5/dist-packages/beautifulsoup4-4.7.1-py3.5.egg (from castero)
Requirement already satisfied: cjkwrap in /usr/local/lib/python3.5/dist-packages/CJKwrap-2.2-py3.5.egg (from castero)
Requirement already satisfied: python-mpv in /usr/local/lib/python3.5/dist-packages/python_mpv-0.3.9-py3.5.egg (from castero)
Requirement already satisfied: python-vlc in /usr/local/lib/python3.5/dist-packages/python_vlc-3.0.4106-py3.5.egg (from castero)
Requirement already satisfied: requests in /usr/local/lib/python3.5/dist-packages/requests-2.21.0-py3.5.egg (from castero)
Requirement already satisfied: soupsieve>=1.2 in /usr/local/lib/python3.5/dist-packages/soupsieve-1.8-py3.5.egg (from beautifulsoup4->castero)
  Ignoring backports.functools-lru-cache: markers 'python_version < "3"' don't match your environment
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.5/dist-packages/certifi-2019.3.9-py3.5.egg (from requests->castero)
Requirement already satisfied: chardet<3.1.0,>=3.0.2 in /usr/local/lib/python3.5/dist-packages/chardet-3.0.4-py3.5.egg (from requests->castero)
Requirement already satisfied: idna<2.9,>=2.5 in /usr/local/lib/python3.5/dist-packages/idna-2.8-py3.5.egg (from requests->castero)
Requirement already satisfied: urllib3<1.25,>=1.21.1 in /usr/local/lib/python3.5/dist-packages/urllib3-1.24.1-py3.5.egg (from requests->castero)
root@26a84df338cf:~# castero
Traceback (most recent call last):
  File "/usr/local/bin/castero", line 11, in <module>
    load_entry_point('castero==0.5.3', 'console_scripts', 'castero')()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 561, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2631, in load_entry_point
    return ep.load()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2291, in load
    return self.resolve()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2297, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python3.5/dist-packages/castero-0.5.3-py3.5.egg/castero/__main__.py", line 9, in <module>
    from castero.database import Database
  File "/usr/local/lib/python3.5/dist-packages/castero-0.5.3-py3.5.egg/castero/database.py", line 7, in <module>
    from castero.episode import Episode
  File "/usr/local/lib/python3.5/dist-packages/castero-0.5.3-py3.5.egg/castero/episode.py", line 246
    f"\cb{self.title}\n" \
                       ^
SyntaxError: invalid syntax
```

**Comment by xgi** [comment]
Should be fixed, thanks!

I accidentally used a Python 3.6 feature when I intend to support 3.5.

**Comment by queltosh** [comment]
Thanks for the fast answer. I am creating a docker image with this project, Its working with version 0.4.2 but if i change it to try the latest comit this is what i get:


```
Traceback (most recent call last):
  File "/usr/local/bin/castero", line 11, in <module>
    load_entry_point('castero==0.5.3', 'console_scripts', 'castero')()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 561, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2631, in load_entry_point
    return ep.load()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2291, in load
    return self.resolve()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2297, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python3.5/dist-packages/castero-0.5.3-py3.5.egg/castero/__main__.py", line 79, in <module>
    main()
  File "/usr/local/lib/python3.5/dist-packages/castero-0.5.3-py3.5.egg/castero/__main__.py", line 15, in main
    database = Database()
  File "/usr/local/lib/python3.5/dist-packages/castero-0.5.3-py3.5.egg/castero/database.py", line 18, in __init__
    self._conn = sqlite3.connect(self.PATH, check_same_thread=False)
sqlite3.OperationalError: unable to open database file
```



The Dockerfile i'm using is:
```
FROM debian:9-slim
RUN apt-get update && apt-get install -yy python3 python3-pip wget git
RUN apt-get clean
RUN git clone https://github.com/xgi/castero
RUN cd /castero && python3 setup.py install
ENTRYPOINT ["castero"]

```

**Comment by xgi** [comment]
Thanks for the dockerfile.

This should be fixed as of d897ddc309c390c4661b477bc99dec6636404ccc. I had to ensure that the $XDG_DATA_HOME/castero path existed before trying to create the database file.

Sorry for the issues!

**Comment by queltosh** [comment]
Now works great. Thanks a lot!
